addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2302,8 +2360,53 @@ export class OdooEditor extends EventTarget {
         this.cleanForSave();
         this.observerActive();
     }
+
+    /**
+     * initialise the provided element to be ready for edition
+     *
+     */
+    initElementForEdition(element = this.editable) {
+        // Flag elements with forced contenteditable=false.
+        // We need the flag to be able to leave the contentEditable
+        // at the end of the edition (see cleanForSave())
+        for (const el of element.querySelectorAll('[contenteditable="false"]')) {
+            el.setAttribute("oe-keep-contenteditable", '1');
```

Comment:
On peut pas avoir des `true` plutôt ? Pour ça et l'autre avec les zws là. Je trouve que ça fait un peu bizarre ces '1' magiques qui se baladent dans le code... ^^'

Reply:
Ben c'est obligé d'être une string, vu que ça va dans un attributs HTML. 
Ça risque d'être ambigus de devoir faire `attribute === 'true'` non ?

Reply:
Bah je trouve ça moins bizarre que `attribute === '1'` personellement ? De toute façon techniquement tu peux juste faire `hasAttribute` en fait, non ? Tu t'en fous de sa valeur. En fait je propose de set l'attribut à `''`, ça va mettre l'attribut mais sans valeur, ainsi après tu dois juste faire `hasAttribute`. Ce sera mieux.

Reply:
ok je vais tester ça.
Attribut sans valeur.

J'espère que ça va pas être cleané par le browser, mais je pense pas.

Reply:
Seems perfect in fact : https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes